sailor/assetcentral/utils.py

```diff
@@ -197,9 +201,14 @@ def quote_if_string(x):
 
         if k in field_map:
             key = field_map[k].their_name_get
+            query_transformer = field_map[k].query_transformer
         else:
             key = k
             not_our_term.append(key)
+            query_transformer = None
+
+        if query_transformer:
+            v = query_transformer(v)
```

Comment:
After discussion with @LeenaPajo we decided not to make the code more generic and confusing that it needs to be. 

Making the query_transformer more generic is not required for the current only use case of transforming timestamps. If we want to make it generic, the above problem comes up, making the code less readable. A default query transformer cannot be defined because of the above problem.

The (only?) downside of this decision is that the query transformer needs to take care of possible quoting.
E.g.: `extended_filters=["abc > 'def'"]` ==> query transformer receives the value `'def'` and needs to take care of the quotes. For the timestamp offset query transformer this is a no-brainer, since pandas will handle it.